### ISSUE TITLE
Issue with "WebKit CSS extensions": needs updating

### ISSUE THREAD

**Issue by nt1m** [mdn/content#13879]
MDN URL: [developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions](https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions)
What information was incorrect, unhelpful, or incomplete?

The "[WebKit extensions](https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions)" page needs to have the properties moved around too. E.g. -webkit-print-color-adjust needs to move from "WebKit-only properties" to "Formerly proprietary properties that are now standard". All the removed properties in 15.4 need to be moved from "WebKit-only properties" to "Deprecated -webkit- properties".

There are probably more examples of properties in the wrong section on that page. Safari 15.4 and previous versions have unprefixed a couple of properties, see: [webkit.org/blog/12445/new-webkit-features-in-safari-15-4](https://webkit.org/blog/12445/new-webkit-features-in-safari-15-4/)

[webkit.org/css-status](https://webkit.org/css-status/) is a good reference.
MDN Content page report details

    Folder: en-us/web/css/webkit_extensions
    MDN URL: [developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions](https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions)
    GitHub URL: [https://github.com/mdn/content/blob/main/files/en-us/web/css/webkit_extensions/index.md](https://github.com/mdn/content/blob/main/files/en-us/web/css/webkit_extensions/index.md?rgh-link-date=2022-03-14T20%3A58%3A59Z)
    Last commit: https://github.com/mdn/content/commit/008f3fb65925484cdb29e8bf0506697096cd0b6a
    Document last modified: 2022-02-01T11:36:33.000Z

**Comment by casaper** [comment]
Hello @nt1m 

A few questions:

- Where are deprecated WebKit CSS extensions in MDN? With a quick go, I couldn't find them.
- standard print-color-adjust is only supported by Safary 15.4, released 2 days ago. Should the vendor prefix go already?

**Comment by nt1m** [comment]
@casaper They're subsections of https://developer.mozilla.org/en-US/docs/Web/CSS/WebKit_Extensions

**Comment by teoli2003** [comment]
>     * standard print-color-adjust is only supported by Safari 15.4, released 2 days ago. Should the vendor prefix go already?
Yes. The browser compatibility section in `print-color-adjust` will have the info for older browsers. People supporting them are used to check them (I think there are event automated tools out there).

**Comment by casaper** [comment]
@nt1m 

I was looking for another page, 😅 sorry about that. Silly of me.

Going to make a PR now to resolve this.

**Comment by casaper** [comment]
@nt1m @teoli2003 

Any chance you guys know what should happen to `-webkit-mask-box-image` and `-webkit-mask-box-image-*` properties?

The WebKit release notes don't seem to mention these,  
and in https://www.w3.org/Style/CSS/all-properties.en.html they are listed amongst old proposed properties.

And CanIUse on the other hand thinks its all there: https://caniuse.com/?search=mask-box-image
